.github/workflows/ci-podman-ubuntu.yml

```diff
@@ -0,0 +1,118 @@
+name: Podman CI Ubuntu
+
+on:
+  push:
+    branches: [ main ]
+    paths-ignore: [ docs/** ]
+  pull_request:
+    branches: [ main ]
+    paths-ignore: [ docs/** ]
+
+jobs:
+  podman-tests:
+    uses: ./.github/workflows/reusable-test.yml
+    with:
+      BUILT_EARTHLY_PATH: "./build/linux/amd64/earthly"
+      BUILD_EARTHLY_TARGET: "+for-linux"
+      RUNS_ON: "ubuntu-latest"
+      BINARY: "podman"
+      SUDO: "sudo"
+    secrets: inherit
+
+  podman-examples-1:
+    # TODO: Figure out how to run multiple Podman instances in parallel with different ports for buildkitd
```

Comment:
I thought that these different CI tasks ran on different, sandboxed machines. No idea why this is happening.

Reply:
That's precisely what I thought as well.
However, no matter what I tried they would run into port conflicts if run in parallel. They might not be as "sandboxed" as we think.
My hypothesis is our old tests ran fine because they all used the same Docker environment (reused buildkitd) while the Podman tests do not reuse the same Podman environment (so the container check fails).